udata_hydra/cli.py

```diff
@@ -252,19 +252,13 @@ async def migrate(skip_errors: bool = False, dbs: list[str] = ["main", "csv"]):
 
 
 @cli
-async def purge_checks(limit: int = 2):
-    """Delete past checks for each resource_id, keeping only `limit` number of checks"""
-    q = "SELECT resource_id FROM checks GROUP BY resource_id HAVING count(id) > $1"
+async def purge_checks(days_until: int = 60):
+    """Delete outdated checks that are more than `days_until` days old"""
     conn = await connection()
-    resources = await conn.fetch(q, limit)
-    for r in resources:
-        resource_id = r["resource_id"]
-        log.debug(f"Deleting outdated checks for resource {resource_id}")
-        q = """DELETE FROM checks WHERE id IN (
-            SELECT id FROM checks WHERE resource_id = $1 ORDER BY created_at DESC OFFSET $2
-        )
-        """
-        await conn.execute(q, resource_id, limit)
+    log.debug(f"Deleting checks that are more than {days_until} days old")
+    await conn.execute(
+        f"""DELETE FROM checks WHERE created_at < now() - interval '{days_until} days'"""
+    )
```

Comment:
Nit: do we want another log showing the number of deleted checks? Would that be useful? Maybe something like
```Deleting 12 outdated checks. 1253 remaining.```

Reply:
(If we get a return value it would be nice to print, I would not start another query to show the number remaining though)

Reply:
I think it's a good idea and could be very helpful. Since it's a query which is not run very often, we can afford it, if it's in the same query.

Reply:
I just updated it. Let me know what you think